includes/definitions/truenas-scale.yaml

```diff
@@ -0,0 +1,27 @@
+os: truenas-scale
+type: storage
+text: TrueNAS-SCALE
+processor_stacked: true
+mib_dir: ixsystems
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+    - { graph: device_storage, text: 'Storage Usage' }
+poller_modules:
+    bgp-peers: false
+    mpls: false
+    ospf: false
+    stp: false
+    wireless: false
+discovery_modules:
+    bgp-peers: false
+    mpls: false
+    stp: false
+    wireless: false
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.8072.3.2.8
+            - .1.3.6.1.4.1.50536.3
+        sysDescr_regex:
+            - '/truenas-scale/i'
```

Comment:
```suggestion
discovery:
    -
        sysDescr_regex:
            - '/truenas-scale/i'
```

Reply:
Good point!  
- I'll remove freenas sysObjectID: .1.3.6.1.4.1.8072.3.2.8 (it has no place in truenas-scale)
- I will leave sysDescr_regex to discover scale 
- I can leave SysObjectID .1.3.6.1.4.1.50536.3 to document that it is the id it has and its shared with CORE (if you don't mind). It will match according to https://docs.librenms.org/Developing/os/Initial-Detection/#discovery-logic